### ISSUE TITLE
Throwing

### ISSUE THREAD

**Issue by DendronAnimats** [cjddmut/Unity-2D-Platformer-Controller#56]
Hi, I would like the player to be able to throw something like a ball at something.  Is this possible in Unity 5+?

**Comment by RussellBloxwich** [comment]
Yes, its very possible. Initialize a new object with a starting vector.
